framework/platform/bpl/uci/cfg/bpl_cfg.cpp

```diff
@@ -157,8 +157,8 @@ int cfg_get_device_info(BPL_DEVICE_INFO *device_info) { return 0; }
 
 int cfg_get_wifi_params(const char iface[BPL_IFNAME_LEN], struct BPL_WLAN_PARAMS *wlan_params)
 {
-    int retVal    = 0;
-    int index     = 0;
+    int retVal = 0;
+    int index  = 0;
```

Comment:
This change should have been part of the commit that updates this function.

Reply:
I'll try not to forget about this when rebasing...